Neos.Flow/Classes/Http/RequestHandler.php

```diff
@@ -90,15 +95,15 @@ public function getPriority()
      */
     public function handleRequest()
     {
-        // Create the request very early so the ResourceManagement has a chance to grab it:
-        $request = Request::createFromEnvironment();
-        $response = new Response();
-        $this->componentContext = new ComponentContext($request, $response);
-
         $this->boot();
         $this->resolveDependencies();
+
+        $request = Request::createFromEnvironment();
```

Comment:
Needed to be moved after `resolveDependencies()` and initialization of ObjectManagement. However, I don't really see any issue with that, since no place in the bootstrap does (and should) need an Http Context with Request and/or Response object.

Reply:
In my change I totally got rid of Request and I would rather prefer that. This whole transition will be breaking, so lets at least use it to clean up as well.